app/src/main/java/com/arctouch/test/vm/ConfigViewModel.kt

```diff
@@ -0,0 +1,21 @@
+package com.arctouch.test.vm
+
+import android.arch.lifecycle.MutableLiveData
+import com.arctouch.test.data.model.Config
+import com.arctouch.test.data.repository.ConfigRepository
+import com.arctouch.test.schedulers.ISchedulerProvider
+import io.reactivex.Observable
+
+class ConfigViewModel constructor(
+        private val configRepository: ConfigRepository,
+        schedulerProvider: ISchedulerProvider
+) : BaseViewModel(schedulerProvider) {
+
+    val config = MutableLiveData<Config>()
```

Comment:
You were using `RxJava Observable` on all layers (Repositories, DataSources...), so why did you change to `LiveData` on the `ViewModel`?

Reply:
To be honest I don't know. I thought it was a recommendation to use live data to communicate between view and view model and rxjava in the repository because livedata is lifecycle aware.